### PR TITLE
Gfrancu/us112978 select outcomes hierarchical

### PR DESCRIPTION
**Pull request by gfrancu**
Using new hierarchical view for select outcomes

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -26703,7 +26772,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#93c20cd9f0c9bfc198d97b434d827e8f05b27974",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#505814e201990646a298d1705c52ba4daf4593c7",
```
Comment (djwester) [inline package-lock.json]: might want to remove these changes - not sure if they are ready to bump the siren-sdk repo yet.

Comment (gfrancu) [reply package-lock.json]: not sure i understand. if commits have gone into master for siren-sdk under the flag of 1.x.x, should these not officially be safe to pull?

Comment (ppaskaris-d2l) [reply package-lock.json]: That's a safe assumption, yeah.

Comment (gfrancu) [reply package-lock.json]: awesome thanks! will do that next time

**Pull request merged**